src/Utilities/ApplePayHelpers.res

```diff
@@ -106,14 +106,25 @@ let startApplePaySession = (
   }
 
   ssn.onshippingcontactselected = shippingAddressChangeEvent => {
+    let currentAmount =
+      paymentRequest->getDictFromJson->getDictFromDict("total")->getString("amount", "0.00")
+    let oldTotal: lineItem = {
+      label: "apple",
```

Comment:
Could you please take these values from the session token response instead of hardcoding it in the SDK?
![Screenshot 2024-10-16 at 5 59 29 PM](https://github.com/user-attachments/assets/7bf3f60e-0f9e-402b-9363-e4eae443fd62)